cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -670,8 +671,9 @@ define(["jquery", "common/js/spec_helpers/ajax_helpers", "common/js/components/u
                             is_prereq: false,
                             "is_time_limited": true,
                             "is_practice_exam": false,
-                            "is_proctored_exam": true,
-                            "default_time_limit_minutes": 150
+                            "is_proctored_exam": false,
```

Comment:
This test structure wasn't doing anything special related to proctored exams, so I changed it to be a regular timed exam (see https://github.com/edx/edx-proctoring/blob/master/edx_proctoring/api.py#L1828-L1843 for how this is handled by the proctoring API) in order to extend the test to cover the new feature.